api/src/main/java/org/apache/cloudstack/api/command/admin/usage/ListUsageRecordsCmd.java

```diff
@@ -145,6 +150,9 @@ public void setUsageId(String usageId) {
         this.usageId = usageId;
     }
 
+    public boolean getOldFormat() {
+        return oldFormat != null && oldFormat;
```

Comment:
this is a way to make the default be false. It should be in the description of the Parameter-annotation.

Reply:
Will fix description.

Reply:
I agree that this default is sensible and I don't think the backwards incompatibility of not using internal ids anymore is going to be very hurtful as the processing of such ids is more intensive than processing uuids using the api. maybe a point of discussion for some, so let's be extra cautious.